packages/components/organisms/f-content-cards/src/components/ContentCards.js

```diff
@@ -233,11 +250,34 @@ export default {
                     callbacks: {
                         handleContentCards: this.metadataContentCards
                     },
-                    loggerCallbacks: {
-                        logger: this.handleLogging(this.$logger)
-                    }
+                    logger: this.$logger,
+                    tags: this.tags
                 });
+
+                this.$logger.logInfo(
+                    `Content Cards (setupMetadata) - BrazeAdapter successfully initialised. Key: (${this.loggingKey})`,
+                    this.$store,
+                    {
+                        data: {
+                            key: this.loggingKey
+                        },
+                        tags: this.tags
```

Comment:
Can we keep the order of keys in the extra data argument consistent between invocations to the various `logger.log*` methods, please? Either change it here as so, to match the others:
```suggestion
                        tags: this.tags,
                        data: {
                            key: this.loggingKey
                        }
```
or ideally, make all invocations have alphabetical order of extra log data - though can accept this is changing every other invocation so happy if you just want to do this one.